host/class/msc/usb_host_msc/include/usb/msc_host_vfs.h

```diff
@@ -16,6 +16,16 @@ extern "C" {
 
 typedef struct msc_host_vfs *msc_host_vfs_handle_t;           /**< VFS handle to attached Mass Storage device */
 
+/**
+ * @brief Format MSC device.
+ *
+ * @param[in] device Device handle obtained from MSC callback provided upon initialization
+ * @param[in] mount_config Mount configuration
+ * @param[in] vfs_handle Handle to MSC device associated with registered VFS
+ * @return esp_err_t
```

Comment:
Here we usually describe the possible return values. The description for the errors should be clear, when you read them. So, for example, for error argument: 
- "Device handle could not be NULL" is better than just the "Wrong argument".

```suggestion
 * @return
 *    - ESP_OK: Format completed  
 *    - ESP_ERR_INVALID_ARG: All arguments must be present and couldn't be NULL
```

Reply:
thank you for suggestion, fixed.